tools/viz.py

```diff
@@ -14,16 +15,46 @@ def __init__(
         ice_thickness: str,
         x: str = "x",
         y: str = "y",
-        time: str = "time",
-        topo_bedrock: str = "topo_bed_rock",
+        x_border: int = 100,
```

Comment:
nit: you could name it `x_pad` and `y_pad` and below use [Dataset.pad](https://docs.xarray.dev/en/latest/generated/xarray.Dataset.pad.html) instead of `.isel`.

Reply:
I could not figure out what you mean, because with pad I only can extend the coordinate, but what I want to do is to crop it (it is the same you are doing in the original nb with `.isel` for selecting a smaller region of the given dataset).

Reply:
Ah yes I was confused by the meaning of `x_border` and `y_border`.

Reply:
yes maybe I should think about a better name

Reply:
renamed it to `x_nr_of_grid_points` and `y_nr_of_grid_points`